### DISCUSSION TITLE
Custom prompt suffix?

### DISCUSSION THREAD

**Discussion by tcb678** [Textualize/rich#1001] [Ideas]
I'd like to create a custom "console" using Rich that asks for input from the user, but I want that input to have a custom prompt_suffix.  It looks as if that isn't supported, and even creating a new Prompt() and changing the suffix doesn't have any impact.  Am I missing something obvious or should this be a feature request?

**Comment by willmcgugan** [comment]
What have you tried?

**Reply by tcb678** [reply]
```from rich.prompt import Prompt
>>> n = Prompt.ask("Hello")
Hello: foo
>>> n = Prompt.ask("Hello", prompt_suffix=">")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: ask() got an unexpected keyword argument 'prompt_suffix'
>>> n = Prompt()
>>> n.prompt_suffix
': '
>>> n.prompt_suffix = ">>> hello > "
>>> n.prompt_suffix = ">>> hello > "
>>> n.prompt_suffix
'>>> hello > '
>>> n.ask("Hi")
Hi: foo
```

**Reply by tcb678** [reply]
I think what I want is a setter on prompt_suffix and the ability to modify prompt_suffix on all Prompt.ask().

**Reply by willmcgugan** [reply]
Try this...

```python
class PyPrompt(Prompt):
    prompt_suffix = ">>> hello > "
```
